indico/modules/events/editing/client/js/management/editable_type/EditableTypeDashboard.jsx

```diff
@@ -25,78 +27,95 @@ export default function EditableTypeDashboard() {
-        >
-          <Checkbox styleName="toolbar-checkbox" toggle />
-          <span styleName="toolbar-label">
-            <Translate>Allow editors to self-assign editables</Translate>
-          </span>
-          <a className="i-button icon-settings">
-            <Translate>Assign</Translate>
-          </a>
-        </Section>
-      </div>
+      {!selfAssignLoading && (
+        <>
+          <div className="action-box">
+            <Section
+              icon="file"
+              label={Translate.string('Submission is not open yet')}
+              description={Translate.string('Start now')}
+            >
+              <a className="i-button highlight icon-list">
+                <Translate>Start now</Translate>
+              </a>
+            </Section>
+            <Section
+              icon="edit"
+              label={Translate.string('Editing is not open yet')}
+              description={Translate.string('Start now')}
+            >
+              <a className="i-button highlight icon-list">
+                <Translate>Start now</Translate>
+              </a>
+            </Section>
+          </div>
+          <div className="action-box">
+            <Section
+              icon="file"
+              label={Translate.string('File types')}
+              description={Translate.string('Configure file types')}
+            >
+              <Link
+                className="i-button icon-settings"
+                to={manageFileTypesURL({confId: eventId, type})}
+              >
+                <Translate>Configure</Translate>
+              </Link>
+            </Section>
+            <Section
+              icon="equalizer"
+              label={Translate.string('Ready for review conditions')}
+              description={Translate.string('Configure conditions for reviewing')}
+            >
+              <Link
+                className="i-button icon-settings"
+                to={manageReviewConditionsURL({confId: eventId, type})}
+              >
+                <Translate>Configure</Translate>
+              </Link>
+            </Section>
+            <Section
+              icon="users"
+              label={Translate.string('Editing team')}
+              description={Translate.string('Configure editing team')}
+            >
+              <a className="i-button icon-mail">
+                <Translate>Contact</Translate>
+              </a>
+              <a className="i-button icon-users">
+                <Translate>Manage team</Translate>
+              </a>
+            </Section>
+            <Section
+              icon="copy1"
+              label={Translate.string('Editable assignment')}
+              description={Translate.string('Assign editors')}
+            >
+              <Checkbox
+                styleName="toolbar-checkbox"
+                toggle
+                checked={selfAssignEnabled}
+                onClick={!selfAssignSaving && toggleSelfAssign}
```

Comment:
![image](https://user-images.githubusercontent.com/12183954/78908774-812c8a00-7a7a-11ea-8d1e-7b657237603c.png)

When `selfAssignSaving` is false and you click the toggle. The component will re-render with `selfAssignSaving` as true and `onClick` will be `!true && toggleSelfAssign = false`

Reply:
Ah it was me suggesting that shortcut 🙈

But you are right, this is a problem here - the code I had in mind when making this suggestion either have something that's `null` and not a boolean before the `&&` part or do something like this:

    onClick={() => !foo && bar()}

Reply:
Yes, the shortcut will work just fine until the re-render. I would suggest either that `() => {}`, which IMO is probably the simplest.
Or converting that value to `null/undefined`, even though it looks bad, can't think of another alternative rn.

`!true && toggleSelfAssign || null`
or
`!selfAssignSaving ? toggleSelfAssign : null